### ISSUE TITLE
How to add a Steam Launch Arguments with stl gui

### ISSUE THREAD

**Issue by Gigas002** [sonic2kk/steamtinkerlaunch#538]
I want to run my games with [`prime-run`](https://github.com/frostworx/steamtinkerlaunch/wiki/PrimeRun) command or argument, but adding line `prime-run` or `USEPRIMERUN=1` or `__NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only __GLX_VENDOR_LIBRARY_NAME=nvidia` to [this arguments list](https://github.com/frostworx/steamtinkerlaunch/wiki/Game-Command-Line-Arguments) takes absolutely no effect (I didn't find a separate option for this in UI and adding the game to `PRIMERUN` collection didn't take any effect either).
The only way to run the game as I want is to add `prime-run` argument into steam interface directly (e.g. `prime-run steamtinkerlaunch %command%`). Am I adding arguments in a wrong way or something? I tried adding the above arguments line into:
1. `Game Menu` (and `Default`) -> `Custom command arguments` line (with `Use custom command` enabled)
2. `Game Menu` (and `Default`) ->`Game command arguments`

**Comment by frostworx** [comment]
yes you're doing it wrong.
I thought [the wiki you](https://github.com/frostworx/steamtinkerlaunch/wiki/PrimeRun) found is clear here:
The option to enable prime run _(called `USEPRIMERUN`)_ can be found "(in and [Game Config](https://github.com/frostworx/steamtinkerlaunch/wiki/Configuration-Files#Game-Config))"
"When enabled the (following) variables are exported automatically" _(those you are trying to enable via game command line arguments - those are no game command line arguments though)_

also:

"Enabling PrimeRun is also possible by simply adding the game to the "PrimeRun" [Steam Collection](https://github.com/frostworx/steamtinkerlaunch/wiki/Steam-Collections)."

So to sum it up you have 3 options to enable PrimeRun for your game:
either
- open the [Game Menu](https://github.com/frostworx/steamtinkerlaunch/wiki/Game-Menu), and cliock the "PrimeRun" checkbox there (whatever it is called in the menu)

or

- open the steamtinkerlaunch game config, and set `USEPRIMERUN` to `1`

or

create a "PrimeRun" [Steam Collection](https://github.com/frostworx/steamtinkerlaunch/wiki/Steam-Collections) and drag&drop  your game into it.

**Comment by Gigas002** [comment]
Thanks for reply. For the reference, I use [latest release,](https://github.com/frostworx/steamtinkerlaunch/releases/tag/v10.0) not a git version of package.

There's no corresponding checkbox in UI for `USEPRIMERUN` option, but I did managed to find it into config file, though changing it's value (in both `default_template.conf` and `game_id.conf`) to `1` has no effect at all. Same's true for collections method.

When using `git` version of package, changing the value in config does takes the desired effect, but somehow `mangohud` stops working at all, though I guess it''s not related.

P.S. Ah, I did found out the `Use PrimeRun` button on `git` version's UI. Guess it wasn't yet implemented in `10.0`.

**Comment by frostworx** [comment]
you can always see in the [Changelog](https://github.com/frostworx/steamtinkerlaunch/wiki/Changelog) when a feature was added.
*closing here*
